### PR TITLE
Add in-memory as an option

### PR DESCRIPTION
**Pull request by gregturn**
Make cassandra configuration conditional, but enabled by default for backwards compatibility.

### CODE REVIEW

**Comment by gregturn** [comment]
This introduces the option to switch off cassandra (critical for PCF) and switch to using the in-memory implementations.

@spinnaker/reviewers PTAL

File: echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/config/SchedulerConfiguration.groovy [hunk 1]
```diff
@@ -44,21 +46,42 @@ import java.util.concurrent.TimeUnit
...
     }
 
     @Bean
+    @ConditionalOnExpression('${spinnaker.cassandra.enabled:true}')
     TriggerDao triggerDao(Keyspace keyspace) {
         new CassandraTriggerDao(keyspace)
     }
 
     @Bean
+    @ConditionalOnExpression('${spinnaker.in-memory.enabled:false}')
```
Comment (tomaslin) [inline echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/config/SchedulerConfiguration.groovy]: change to inMemory to be consistent with all other projects

**Comment by tomaslin** [comment]
LGTM after changing property

**Comment by gregturn** [comment]
Done.

**Comment by tomaslin** [comment]
why do this instead of cassandra.embedded : true

**Comment by gregturn** [comment]
Huh? This has nothing to do with Cassandra, embedded or not.

**Comment by tomaslin** [comment]
the embedded cassandra functionality would work the same as the in-memory implementation of the scheduler

**Pull request merged**